scripts/upgrade

```diff
@@ -179,15 +175,10 @@ chown -R $app: $final_path
 #=================================================
 # SETUP SSOWAT
 #=================================================
-ynh_print_info "Upgrading SSOwat configuration..."
 
-# Make app public if necessary
-if [ $is_public -eq 1 ]
-then
-	# unprotected_uris allows SSO credentials to be passed anyway
-	ynh_app_setting_set $app unprotected_uris "/"
-	# protect admin area
-	ynh_app_setting_set $app protected_uris "/admin"
+if ynh_exec_warn_less ynh_permission_create --permission "admin"; then
```

Comment:
@alexAubin Is it the only way to avoid an error if the permission already exists?

Reply:
https://github.com/YunoHost/yunohost/pull/840